### ISSUE TITLE
Performance requirements

### ISSUE THREAD

**Issue by saschagrunert** [kubernetes-sigs/cri-o#2008]
During the writing of the unit tests for cri-o I was wondering if there are any special performance requirements for the overall project. Do we have currently any measures / tests / comparisons and is there a need for it (also in relation to other runtimes)? If yes, I could imagine that we can work on that after #1971 is done. What do you think?

**Comment by vrothberg** [comment]
Sounds good to me. Currently, we are working on improvements in containers/image and constainers/storage to speed up image distribution, make use of blob caches etc. Some things have already been merged into CRI-O (e.g., faster gzip compression, parallel pull from registries) other are still on the TODO list (e.g., parallel writing/committing to the storage).

Maybe #1971 will reveal more things to be improved, which would be nice.

**Comment by rhatdan** [comment]
@saschagrunert Do we still need this issue open?

**Comment by saschagrunert** [comment]
No, I will open a dedicated issue If I find something specific.
